### PR TITLE
BL-962_descriptive_Paragraphs_for_visualizations

### PR DESCRIPTION
**Pull request by TophertheCoder**
## Description
We added a descriptive paragraph for the tech stack by role visualization.

## Jira Ticket
[Jira Ticket.](https://bloomtechlabs.atlassian.net/browse/BL-962?atlOrigin=eyJpIjoiMjk0NTA3NjVkZWMyNGMyYmE3OTRmOWZkZTNiODM5Y2IiLCJwIjoiaiJ9)

## Type of Change
- [x] New feature

## Checklist
- [x] My code follows PEP8 style guide
- [x] I have removed unnecessary print statements from my code
- [x] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no errors
- [x] No commented-out code
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

## Loom Video
[Loom video](https://www.loom.com/share/73af376d0d2b4d9ebe3bc8868a413ecc)
[Loom video update](https://www.loom.com/share/786de2afe121480a8ea6ad2ded41a420)

### CODE REVIEW

File: app/graphs.py [hunk 1]
```diff
@@ -42,4 +42,7 @@ def df_tech_stack_by_role(database: MongoDB) -> DataFrame:
     }, {"tech_stack": True}))
     mentors = mentors.explode(column="tech_stack")
     mentors["role"] = "Mentor"
-    return pandas.concat([mentees, mentors])
+    desc_para_final = "This graph shows the ratio of Mentors to Mentees. " \
```
Comment (woodbenwood) [inline app/graphs.py:45]: A great start to something that will evolve incrementally as we continue to work with Web on implementation. It's necessary that this come from the DS side, so we retain the ability to adjust the description as parameters and visualizations change.

**Review by woodbenwood — APPROVED** [review]

**Review by Austin-T-Johnson — APPROVED** [review]
Great job implementing this on the DS side, having it here makes it easily changeable as the data might also be changing in the future. Clean code very readable.

**Review by ezra-casas — APPROVED** [review]
This s great stuff, yall doing wonders here . Luckily, I studied some python prior to entering this bootcamp :)

**Review by Nicholas-R-Payne — APPROVED** [review]
Great job with adding this!

**Review by MaxT6 — APPROVED** [review]
This is a great addition to making the graphs for response, especially if we add other graphs in the future.

**Pull request merged**